### PR TITLE
Fixes #32

### LINKED ISSUE #32: Fix error and note error in RDS
`https://www.reddit.com/r/todayilearned/comments/gzwlp6/til_6_years_after_resigning_nixon_testified_on` returns 
```"error":"Unable` to obtain infomation from the website.``` 
but this error is not put into the status code and the original website and claim are not saved in RDS

claim: `6 years after resigning, Nixon testified on behalf of former FBI assistant director Mark Felt at Felt's own trial, and gave money to Felt's defense fund. In 2005 Felt revealed he had been "Deep Throat", Bob Woodward's source while breaking the Watergate scandal that led to Nixon's resignation`

### PR DESCRIPTION
**Pull request by connorjoleary**
Now all responses return at least the original link and claim, reddit can be read from, and the start of a notebook for data analytics.

### CODE REVIEW

**Comment by connorjoleary** [comment]
#32

File: backend/aws/rds/drop_notebook_output.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import json
+
+nb = sys.stdin.read()
+
+json_in = json.loads(nb)
+nb_metadata = json_in["metadata"]
+suppress_output = False
+if "git" in nb_metadata:
+    if "suppress_outputs" in nb_metadata["git"] and nb_metadata["git"]["suppress_outputs"]:
+        suppress_output = True
```
Comment (deadmau6) [inline backend/aws/rds/drop_notebook_output.py]: You could remove the `if` statements and instead do:
`suppress_output = nb_metadata.get("git", {}).get("suppress_outputs")`

File: backend/main.py [hunk 2]
```diff
@@ -11,24 +11,28 @@
...
 @app.route('/api/v1/deep_cite', methods=['GET', 'POST'])
 def deep_cite():
+# def deep_cite(claim, link):
     content = request.get_json()
 
     try:
         claim = sanitize_claim(content['claim'])
         link = sanitize_link(content['link'])
     except Exception as e:
-        return jsonify({'error': 'Error 505: HTTP Verison Not Supported, hacker' })
```
Comment (deadmau6) [inline backend/main.py:20]: lol, noice

File: backend/main.py [hunk 3]
```diff
@@ -11,24 +11,28 @@
...
-        full_pre_json = {'results': tree.get_best_path()}
+        full_pre_json['results'] = tree.get_best_path()}
+
+    # TODO: If one website fails then we shouldn't return only an error for everything
     # handles exceptions that arise
     except Exception as e:
         if check_instance(e):
             full_pre_json['error'] = str(e)
         else:
+            # TODO: this should be better
```
Comment (deadmau6) [inline backend/main.py]: Hmm, agreed this should be an issue

File: extension/manifest.json [hunk 4]
```diff
@@ -14,14 +14,12 @@
...
-    "contextMenus",
-    "storage",
-    "tabs"
+    "storage"
   ],
   "content_scripts": [
     {
-      "matches": ["<all_urls>"],
-      "js": ["js/jquery-3.2.1.min.js","js/content.js"]
+      "matches": ["https://jzvkkf7p6d.execute-api.us-east-2.amazonaws.com/*"],
```
Comment (deadmau6) [inline extension/manifest.json:21]: Is this essentially running the production server?

Comment (connorjoleary) [reply extension/manifest.json:21]: Yah, although die to lack of funds it is just running the server

File: backend/main.py (hunk 3 above)
Comment (connorjoleary) [reply backend/main.py]: Good call. I'll add it

**Pull request merged**